addons/lunch/views/lunch_product_views.xml

```diff
@@ -57,7 +57,8 @@
         <field name="arch" type="xml">
             <xpath expr="//tree" position="attributes">
                 <attribute name="js_class">lunch_list</attribute>
-            </xpath>
+                <attribute name="create">false</attribute>
+            </xpath>        
```

Comment:
```suggestion
            </xpath>
```

Reply:
@kba-odoo I have removed the trailing spaces and added a commit. Can you please review my PR again?
TIA.

Reply:
@odooaktiv could you squash the commits - it's not necessary to have a commit just for the removal of whitespaces - and also change the first line of  the message to be something like: `[FIX] lunch: hide create button on the list view` to be consistent with our [guidelines]

[guidelines]: https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#commit-message-structure

Reply:
Hello Kevin @kba-odoo , I have updated and made necessary changes. Can you please Review the PR?